core/src/components/badge/badge.ionic.scss

```diff
@@ -174,3 +175,23 @@
   min-width: globals.$ion-scale-400;
   height: globals.$ion-scale-400;
 }
+
+// Badge Inside Tab Button
+// --------------------------------------------------
+
+:host([vertical].in-tab-button) {
+  position: relative;
+
+  min-width: $badge-ionic-in-tab-button-min-width;
```

Comment:
Remove this since the badge size should be modified directly to the tag like ` <ion-badge vertical="top" color="warning" size="xxsmall">999</ion-badge>`

```suggestion
```

Reply:
According figma, once inside tab button there is only 1 size, that's why this is needed.

Reply:
This is needed to change the min-width inherit in order to fit design needs, specially when badge is empty.